src/sync/build.ts

```diff
@@ -25,17 +27,27 @@ const getTransformedBuilds = async (workflowRun, gitHubInstallationClient, logge
 export const getBuildTask = async (
 	logger: Logger,
 	gitHubInstallationClient: GitHubInstallationClient,
-	_jiraHost: string,
+	jiraHost: string,
 	repository: Repository,
-	cursor: string | number = 1,
-	perPage?: number
+	cursorOrigStr: string | number = 1,
```

Comment:
Nit: I can see we're now using cursor and perPage in the function block but is there any other names we could use here? perPageOrig, cursorOrigStr, cursorOrig aren't super clear and it gets a little hard keeping track of what's what. I'm okay with letting this through now because it's an experiment but would probably be good to update these for readability sakes if we decide to keep this in the code.

Reply:
I'm open for suggestions :)